### PR TITLE
Ensure training and inference use raw counts

### PR DESCRIPTION
**Pull request by ShinDongWoon**
## Summary
- feed unnormalised wide matrices into the sliding-window dataloaders so the model and loss operate on raw counts while still recording scaler metadata for artifacts
- adjust inference to skip scaling, producing clipped forecasts directly on the original count scale

## Testing
- python -m src.timesnet_forecast.train --config configs/default.yaml --override train.device=cpu train.epochs=1 train.batch_size=2 train.num_workers=2 train.pin_memory=false train.persistent_workers=false train.prefetch_factor=2 train.cuda_graphs=false train.compile=false train.accumulation_steps=1 train.amp=false train.val.strategy=holdout train.val.holdout_days=10 model.input_len=7 model.pred_len=2 model.d_model=32 model.n_layers=1 model.k_periods=1 model.kernel_set=[3] model.pmax_cap=60 model.min_period_threshold=1 train.grad_clip_norm=0 train.use_loss_masking=false
- pytest

------
https://chatgpt.com/codex/tasks/task_e_68ca2d7f97908328b5e8530f333d5732

### CODE REVIEW

**Comment by ShinDongWoon** [comment]
@codex review
